aequilibrae/paths/traffic_assignment.py

```diff
@@ -482,13 +482,23 @@ def info(self) -> dict:
         """
 
         classes = {}
+
         for cls in self.classes:
+            uclass = {}
+
             if len(cls.matrix.view_names) == 1:
-                classes[cls.graph.mode] = {nm: np.sum(cls.matrix.matrix_view[:, :]) for nm in cls.matrix.view_names}
+                uclass['matrix_totals'] = {nm: np.sum(cls.matrix.matrix_view[:, :]) for nm in
+                                           cls.matrix.view_names}
             else:
-                classes[cls.graph.mode] = {
-                    nm: np.sum(cls.matrix.matrix_view[:, :, i]) for i, nm in enumerate(cls.matrix.view_names)
-                }
+                uclass['matrix_totals'] = {nm: np.sum(cls.matrix.matrix_view[:, :, i]) for i, nm in
+                                           enumerate(cls.matrix.view_names)}
+            uclass['network mode'] = cls.graph.mode
+            uclass['Value-of-time'] = cls.vot
+            uclass['PCE'] = cls.pce
+            if cls.fixed_cost_field:
+                uclass['Fixed cost field'] = cls.fixed_cost_field
+                uclass['Fixed cost multiplier'] = cls.fc_multiplier
+            classes[cls.__id__] = uclass
```

Comment:
Should we make sure that class names are unique and raise a helpful error otherwise? Or is this already part of the code and therefore does not show up in the changes here?